habana/pytorch/buildspec-1-10.yml

```diff
@@ -1,7 +1,7 @@
 account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
```

Comment:
We need to follow the buildspec naming convention for this as well

Reply:
Also, when we do that, I guess we will have 2 buildspec files made, 1 for synai1.2 and the other for synai1.3.